### ISSUE TITLE
Repeating posts list

### ISSUE THREAD

**Issue by noplanman** [humanmade/repeatable-posts#11]
It might make sense to add a new link to the posts lists to easily display all repeating posts in one list.
When handling a whole bunch of posts, this makes it easier to manage repeatable ones.

Something like this:
![repeating-posts](https://cloud.githubusercontent.com/assets/9423417/10406126/10b65474-6ede-11e5-919d-f05f9f1b82a0.png)

**Comment by willmot** [comment]
I had considered this and I generally like this idea.

One question I had, would this only show *repeating* posts (that is the original) or would it also show all the *repeat* posts as well.

**Comment by noplanman** [comment]
Maybe show them all by default and add a filter to make it optional (all, only repeatable, only repeat)?
If someone has so many repeatable posts that it becomes difficult to navigate, they can then use the filter to narrow it down.

Also, in the same way that the *repeat* posts link back to the original *repeatable* one, maybe we can save the post IDs of the *repeat* posts to the *repeatable* one? This way, it would be easy to find all *repeat* posts and have an overview of them.
What do you think?

**Comment by willmot** [comment]
> Also, in the same way that the repeat posts link back to the original repeatable one, maybe we can save the post IDs of the repeat posts to the repeatable one? This way, it would be easy to find all repeat posts and have an overview of them.

I think we could achieve this by querying by `post-parent` as each repeat post has it's `parent` set to the original repeating post.

**Comment by noplanman** [comment]
Yes you are absolutely right! That makes it much easier :+1: 

So if we were to implement it, the question would be:
How do we display it in the publish meta box?
Or would it make sense to create a custom meta box for the repeatable functionality?

I really like the simple dropdown as it is at the moment and it also fits where it is.

**Comment by willmot** [comment]
Agree it would be nice to keep this in the same publish box. Maybe something like
![screen shot 2015-10-12 at 11 47 48](https://cloud.githubusercontent.com/assets/308507/10426328/15c18b00-70d7-11e5-9d07-de8115fa2977.png)

We'd then hide that text when you click edit and show it again when the dropdown is hidden.

**Comment by noplanman** [comment]
Me gusta! :smiley: 
The link would then open a page which has the filters set to only display the children?

**Comment by willmot** [comment]
Yep, I guess it would like to this page ![](https://cloud.githubusercontent.com/assets/9423417/10406126/10b65474-6ede-11e5-919d-f05f9f1b82a0.png) but with the `post-parent` query param also set so we're just seeing children.

**Comment by dashaluna** [comment]
@willmot I've added table view links:

<img width="762" alt="screenshot 2017-12-20 15 47 54" src="https://user-images.githubusercontent.com/2185972/34253294-e37c2a44-e63e-11e7-8942-ef1b7e7ad3ae.png">

I wanted to discuss what other view related functionality we'd want to do here? Perhaps now, when viewing `Repeating` posts we can add link to `View # repeat posts` for it, ex screenshot:

<img width="779" alt="screenshot 2017-12-21 11 07 09" src="https://user-images.githubusercontent.com/2185972/34253394-3d7dcb4c-e63f-11e7-9e66-5e7ad4ef94e0.png">

We can then display them via ajax or reload the page with only that specific `repeating` post with all its `repeat` posts on the same screen as child posts (like in the Pages hierarchical view).

Let me know your thoughts :)

**Comment by willmot** [comment]
Left a comment on #31, I think adding in a View # repeats is a great idea for an enhancement, go for it 👍

**Comment by dashaluna** [comment]
@willmot cool, thanks for letting me know. I'll carry on! I will do it as separate branch & PR, so that #31 can be merged and build on :)
